openapi.yml

```diff
@@ -32,71 +55,98 @@ paths:
                 type: array
                 items:
                   $ref: "#/components/schemas/OrderFull"
-  /api/v1/nonce:
+  /api/v1/fee:
     get:
-      summary: The nonce to pick for a new order.
-      description: The best nonce to pick depends on how much time there is left in the current settlement according to the operator. This time is token-pair specific which is why the sell and buy tokens must be supplied.
+      description: |
+        The fee that is charged for placing an order. 
+        The fee is described by a minimum fee - in order to cover the gas costs for onchain settling - and
+        a feeRatio charged to the users for using the service. 
       parameters:
         - name: sellToken
           in: query
           schema:
             $ref: "#/components/schemas/Address"
-        - name: buyToken
-          in: query
-          schema:
-            $ref: "#/components/schemas/Address"
       responses:
         200:
-          description: the nonce
+          description: the tip
           content:
             application/json:
               schema:
-                $ref: "#/components/schemas/Nonce"
+                $ref: "#/components/schemas/FeeInformation"
+        400:
+          description: sellToken not existent
 components:
   schemas:
     Address:
       description: Ethereum 40 byte address encoded as a hex without `0x` prefix.
       type: string
       example: "6810e776880c02933d47db1b9fc05908e5386b96"
     TokenAmount:
-      description: Amount of a token. uint112 encoded in decimal.
+      description: Amount of a token. uint256 encoded in decimal.
       type: string
       example: "1234567890"
-    Nonce:
-      description: Nonce of the batch in which the order is valid. uint32.
-      type: integer
-    OrderNew:
+    FeeInformation:
```

Comment:
@anxolin @W3stside This is our newest proposal: we provide you with a feeRatio and a minimalFee.

Then the logic for calculating the fee will be Round(max(feeRatio*sellAmount, minimalFee)). Wdyt?

Reply:
Is this fee ration independent from the protocol fee? Or is meant to be used to replace the protocol fee?

We used to have 0.1% protocol fee, and the tip was for paying gas. My question is, we don't have any hard requirement anymore is just up to the operator to tell his clients to pay what he says?

Reply:
WE want to go with [this](https://gnosisinc.slack.com/archives/C8YGC05NG/p1606824110024700).  Yes, there will only be one fee. This fee is signed off by the user and will be Round(max(feeRatio*sellAmount, minimalFee))

Reply:
Ok 👌
I guess for being more specific, we'll round up to the next wei in the sell token currency.